File: calcolo kwh/ calcoloKWH.c
#include <stdio.h>
#include <stdlib.h>

int main(){

	int numerokwh;
  float impostabolletta, costomedio;

  printf("inserire il numero di kwh: ");
  scanf("%d", &numerokwh);

  if (numerokwh<=500)
  {
    impostabolletta = numerokwh*0.20;
  }

  else if (numerokwh>500 && numerokwh<=1000)
  {
    // 700 => 500*0.20 + 200*0.05
    impostabolletta = 500*0.20+(numerokwh-500)*0.05;
  }

  else if (numerokwh>1000)
  {
    impostabolletta = 500*0.20+500*0.05+(numerokwh-1000)*0.08;
  }

  costomedio = impostabolletta/numerokwh;
  
  printf("Il consumo mensile viene %f", impostabolletta);
  printf("Il costo medio viene %f", costomedio);

  return 0;

}

File: calcolo kwh/calcolo kWh.py
numerokWh = int(input("Inserisci il numero di kWh: "))

if numerokWh <= 500:
    prezzo = numerokWh * 0.2

elif numerokWh <= 1000 and numerokWh > 500:
    # 700 => 500*0.20 + 200*0.05
    prezzo = 500*0.2 + (numerokWh-500)*0.05

else:
    # 1200 => 500*0.20 + 500*0.05 + 200*0.08
    prezzo = 500*0.2 + 500*0.05 + (numerokWh-1000)*0.08


costomedio = prezzo/numerokWh
print("Il costo medio è: ", costomedio)
print("Il prezzo totale è: ", prezzo)